copier/errors.py

```diff
@@ -120,6 +130,37 @@ class MultipleYieldTagsError(CopierError):
     """Multiple yield tags are used in one path name, but it is not allowed."""
 
 
+class TaskError(UserMessageError, subprocess.CalledProcessError):
```

Comment:
Just semantics, but I'd prefer changing the order of the base classes because semantically `TaskError` is rather a subclass of `CalledProcessError`.